README.md

```diff
@@ -1,45 +1,69 @@
 About this container
 ---
-[![Docker Build Status](https://img.shields.io/docker/build/cturra/ntp.svg)](https://hub.docker.com/r/cturra/dropbox/)
-[![Docker Pulls](https://img.shields.io/docker/pulls/cturra/ntp.svg)](https://hub.docker.com/r/cturra/dropbox/)
-[![Apache licensed](https://img.shields.io/badge/license-Apache-blue.svg)](https://raw.githubusercontent.com/cturra/docker-dropbox/badges/LICENSE)
+[![Docker Build Status](https://img.shields.io/docker/build/nicoinn/docker-ntp.svg)](https://hub.docker.com/r/nicoinn/docker-ntp/)
+[![Docker Pulls](https://img.shields.io/docker/pulls/nicoinn/docker-ntp.svg)](https://hub.docker.com/r/nicoinn/docker-ntp/)
+[![Apache licensed](https://img.shields.io/badge/license-Apache-blue.svg)]
```

Comment:
Probably don't want to change these.

Reply:
This commit and subsequent ones were not part of the pull request for a reason...

In here, please review the pull request and not any of my further changes that are intended for a different purpose!
(but feel free to grab something if you think it's useful! :) )